packages/admin/admin/src/dateTime/DatePickerField.tsx

```diff
@@ -0,0 +1,14 @@
+import { type FieldRenderProps } from "react-final-form";
+
+import { Field, type FieldProps } from "../form/Field";
+import { Future_DatePicker as DatePicker, type Future_DatePickerProps as DatePickerProps } from "./DatePicker";
+
+const FinalFormDatePicker = ({ meta, input, ...restProps }: DatePickerProps & FieldRenderProps<string, HTMLInputElement>) => {
+    return <DatePicker {...input} {...restProps} />;
+};
```

Comment:
We should export this, since we do it for all other *Field components as well. What do you think?

Reply:
Since we have the Field-Component I don't see a need for this, so unless we have an actual use-case I'd prefer not to export it. 

Also, for other Final Form components we have this info, which seems like an [alternative deprecation warning](https://github.com/vivid-planet/comet/blob/main/packages/admin/admin-date-time/src/datePicker/FinalFormDatePicker.tsx#L7-L11) 😄 

```ts
/**
 * Final Form-compatible DatePicker component.
 *
 * @see {@link DateField} – preferred for typical form use. Use this only if no Field wrapper is needed.
 */
```

Reply:
IMO this is unnecessary and would never be used, but I started a discussion internally 😄 

> So we should either remove all FinalForm*-variants or we should export it for all components.

I'd just not add new Final-Form variants of components, but leave the existing ones to avoid braking changes (or maybe deprecate them)

Reply:
Discussed this internally and haven't come to a decision yet. We will merge this without the export for now.